_data/glossary/customresourcedefinition.yaml

```diff
@@ -0,0 +1,10 @@
+name: Custom Resource Definition
+aka: Third Party Resource
+tags:
+full-link: docs/tasks/access-kubernetes-api/migrate-third-party-resource/
+- fundamental
+- operation
+short-description: >
+  A custom resource definition is the representation of a customization of a particular Kubernetes installation.
+long-description: >
+  A resource definition that defines the endpoint in the Kubernetes API that stores a collection of API objects of a certain kind that is specified.
```

Comment:
This suggests that you need to add a CRD to be able to call an existing endpoint. But in fact you write a CRD to create custom resources (so that you don't have to write a whole custom K8s API server). If you want to do some more research, here's a great blog post by one of my Heptio colleagues: https://blog.heptio.com/an-introduction-to-extending-kubernetes-with-customresourcedefinitions-76deb675b27a
This is a super tricky term and concept, so bonus points for tackling it!